README.md

```diff
@@ -41,8 +41,11 @@ $ docker run \
   --interactive \
   --tty \
   --volume $(PWD):/src \
-  kubernetes-hugo:latest \
-  hugo
+  -p 1313:1313 \
+  kubernetes-hugo \
```

Comment:
We should be pegging this on a version, and probably not just `latest`, IMO.

Reply:
We could use `$(HUGO_VERSION)` as the version. Does that sound reasonable?

Reply:
I'm not sure that versioning the image makes much sense in this case, as the built image contains the entire contents of the repo (which is highly variable). In this case, `latest` just means "using the up-to-date contents of the repo" rather than corresponding to, say, a specific version of Go or Hugo.

In general, I feel like image versions make sense only when images are meant to be publicly shared via a registry. When they're intended only as convenient vessels for local build artifacts and processes, a versioning scheme becomes unnecessary IMHO.

Reply:
@lucperkins that makes sense to me. I'm happy keeping it as it is if @mistyhacks agrees. :)

Reply:
Another possibility, however, would be to create versioned images based on `HUGO_VERSION`, e.g. `kubernetes-hugo:0.43.0`, that use the website repo as an external volume rather than including the repo inside the image. I'd be happy to make that change if y'all think that's a better approach.

Reply:
If so, I'd say merge this first to make sure the "make API" is where we want it to be and then I can modify what the commands do.